### DISCUSSION TITLE
Inquiry about Applying ComfyUI's iPadPutar in Krita

### DISCUSSION THREAD

**Discussion by zyakusima** [Acly/krita-ai-diffusion#431] [General]
Hello everyone,

I'm using Krita and I'm having trouble applying ComfyUI's iPadPutar. Specifically, I can't figure out how to rotate images using iPadPutar. I've searched through Krita's documentation and online resources, but I couldn't find detailed information.

I'm seeking advice or guidance on how to use ComfyUI's iPadPutar feature in Krita. If anyone knows how to correctly apply iPadPutar or the necessary steps, settings, or installation methods for plugins, I would greatly appreciate your help.

Additionally, any information about the versions of Krita and ComfyUI, as well as other relevant details, would be helpful in providing appropriate advice.

Thank you very much.

Best regards,

**Comment by ETCore7** [comment]
What is "iPadPutar"? Google only shows your question here plus unrelated IPad things.
